package/package_psoc_index.template.json

```diff
@@ -26,16 +26,16 @@
                     "toolsDependencies": [
                         {
                             "packager": "infineon",
-                            "name": "mtb-tools",
-                            "version": "3.2.0.16028"
+                            "name": "ModusToolbox",
```

Comment:
The previous version is more compliant with other packages naming and folder structure conventions. What impedes us to keep it like this?
I will check while reworking the packages, because we still don´t have a consistent naming for packages mtb for different os, versioning and urls.

Reply:
Do you mean using the name mtb-tools instead of ModusToolbox? This is not a problem... I can change it now

Reply:
Perfect 👍